lib/components/Editor/CustomExtensions/Image/ExtensionConfig.js

```diff
@@ -99,16 +99,15 @@ const ImageExtension = Image.extend({
   },
 });
 
-const upload = (file, url) =>
-  new Promise((resolve, reject) => {
-    const uploader = new DirectUpload(file, url);
-
-    uploader.create((error, { blob_url: imageUrl }) => {
-      if (error) return reject(error);
-
-      return resolve(imageUrl);
-    });
-  });
+const upload = async (file, url) => {
+  const uploader = new DirectUpload({ file, url });
+  try {
+    const { blob_url: imageUrl } = await uploader.create();
+    return Promise.resolve(imageUrl);
+  } catch (error) {
+    return Promise.reject(error);
```

Comment:
Similar to comment above, just throw the error and it automatically becomes the rejected value of the promise returned by this function. But in this case, I don't think we need to catch it just to throw it again. just remove try/catch and the error, if occured inside this function will be thrown directly.